implementations/rust/ockam/ockam_transport_tcp/src/portal/portal_message.rs

```diff
@@ -14,6 +14,9 @@ pub enum PortalMessage<'de> {
     /// or from the target to the Inlet was dropped
     Disconnect,
     /// Message with binary payload and packet counter
+    // TODO: Add route_index. May not be as important as for eBPF portals, as regular portals
```

Comment:
Do we actually need the `route_index` when we already have a generic incremental counter?

Reply:
Hm, maybe we could reuse it for that purpose as well. I'm not working on that any time soon though